ReleaseNotes/RN170928.md

```diff
@@ -1,16 +1,16 @@
 ﻿## 170928 
  
 ### Feature updates 
-* Migrated AzSDK alerts to use the new Azure PG alert framework which offers more powerful alert targeting using 'Action Groups'. The older mechanism of activity alerts has been deprecated (by PG). (See next point about how we will migrate everyone to the new approach.) 
-* Added a new cmdlet (Update-AzSDKSubscriptionSecurity) to serve as a single command to ensure all important artifacts such as Alerts, ASC, ARM Policy, RBAC and Continuous Assurance runbook can be updated in one go to match the latest. This will allow teams to keep their subscriptions in sync with the latest security features of Azure and also stay aligned with CSE-wide policy/config changes.  
-* (Preview) Continuous Assurance now supports a 'central scan' mode. This will enable central teams to setup CA in a 'master subscription' and scan an entire group of target subscriptions. Please contact us if you'd like to try this out. 
-* A "Rationale" field has been added to all the 260+ controls across AzSDK. This field covers the "why" aspect of the control and helps a user understand security risks better. 
-* The "fix controls" feature has been extended to most of the controls for key services (AppService, Azure SQL DB, Storage, VMs, etc.). More services will be covered in future sprints. 
+* Migrated AzSDK alerts to use the [new Azure PG alert framework](https://azsk.azurewebsites.net/01-Subscription-Security/Readme.html#azsk-subscription-activity-alerts-1) which offers more powerful alert targeting using 'Action Groups'. The older mechanism of activity alerts has been deprecated (by PG). (See next point about how we will migrate everyone to the new approach.) 
+* Added a new cmdlet [(Update-AzSDKSubscriptionSecurity)](https://azsk.azurewebsites.net/01-Subscription-Security/Readme.html#update-subscription-security-baseline-configuration) to serve as a single command to ensure all important artifacts such as Alerts, ASC, ARM Policy, RBAC and Continuous Assurance runbook can be updated in one go to match the latest. This will allow teams to keep their subscriptions in sync with the latest security features of Azure and also stay aligned with CSE-wide policy/config changes.  
```

Comment:
Use azsk version of Update-AzSDKSubscriptionSecurity

Reply:
Yes ,In Release Note we have AzSDK because release note refer to AzSDK module that's why I kept text as it is but link is pointed to AzSK.
Please reply if I need to update text.